.vscode/settings.json

```diff
@@ -6,7 +6,19 @@
     "cmake.configureSettings": {
         "POWER_GRID_MODEL_SANITIZER": 1
     },
-    "testMate.cpp.test.executables": "${command:cmake.buildDirectory}/bin/power_grid_model_*_tests",
+    "testMate.cpp.test.executables": "${command:cmake.buildDirectory}/bin/power_grid_model_*_tests.exe",
+    "testMate.cpp.debug.configTemplate": {
+        "type": "cppvsdbg",
```

Comment:
just to double-check: `cppvsdbg` exists on all platforms?

Reply:
This is a default config for ms-vscode.cpptools extension of vscode. Refer https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/configuration/debug.configTemplate.md

Check the remarks in the end of the page for the behavior of "type".

Reply:
perfect!